packages/http/test/index.js

```diff
@@ -509,13 +509,50 @@ describe('post', () => {
       return body;
     });
 
+    testServer
+      .post('/api/csv-reader')
+      .times(3)
+      .reply(200, function (url, body) {
+        console.log('thanks for the message!', body);
+        return body;
+      });
+
     testServer
       .post('/api/fake-custom-success-codes')
       .reply(302, function (url, body) {
         return { ...body, statusCode: 302 };
       });
   });
 
+  it('should make an http request from inside the parseCSV callback', async function () {
```

Comment:
@taylordowns2000 just to bring you up to speed on this, here's what your http test looks like now.

Is this all as you expect?

Reply:
@mtuchi is all good with this! thanks joe!